src/main/java/org/venice/beachfront/bfapi/services/JobPoller.java

```diff
@@ -191,34 +192,38 @@ public void processJobStatus(Job job, StatusMetadata status) {
 					jobService.createDetection(job, geometry);
 					// Finally, mark the Job as successful
 					job.setStatus(Job.STATUS_SUCCESS);
+					
+					// mark the log as a success and how long it took to complete
 					piazzaLogger.log(
-							String.format("Successfully recorded Detection geometry for Job %s and marking as Success.", job.getJobId()),
+							String.format("Successfully recorded Detection geometry for Job %s and marking as Success. Job completed in %d ms", job.getJobId(), 
```

Comment:
Can we use some Joda time here? I feel like milliseconds is not important here.

Something like: 

new Duration(createdOn, now).getStandardMinutes()

Minutes will make for much more readable logs , I feel like.

Reply:
You can put this code in one place at the top, before the switch on statuses.